android/app/src/androidTest/kotlin/coop/polypoly/polypod/PodApiTestDouble.kt

```diff
@@ -2,20 +2,25 @@ package coop.polypoly.polypod
 
 import android.webkit.WebView
 import androidx.test.core.app.ApplicationProvider
+import coop.polypoly.polypod.info.Info
+import coop.polypoly.polypod.network.Network
 import coop.polypoly.polypod.polyIn.PolyInTestDouble
 import coop.polypoly.polypod.polyNav.PolyNav
 import coop.polypoly.polypod.polyOut.PolyOutTestDouble
 
 class PodApiTestDouble(
     override val polyOut: PolyOutTestDouble,
-    override val polyIn: PolyInTestDouble
+    override val polyIn: PolyInTestDouble,
+    webView: WebView
 ) : PodApi(
     polyOut,
     polyIn,
     PolyNav(
-        WebView(ApplicationProvider.getApplicationContext()),
+        webView = webView,
```

Comment:
I am a bit confused about why you're doing these changes in the scope of what is the reenablement of instrumentation tests. Are they related, or you just happened to do it at the same time?

Reply:
The tests didn't actually compile, since we never ran them but kept changing the code base, so he first had to fix these issues.